scripts/kubernetes/rabbit_deployment.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: funcatron-rabbit
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: funcatron-rabbit
+    spec:
+      containers:
+        - name: rabbit
+          resources:
+            requests:
+              cpu: 0.5
+              memory: 256Mi
+          imagePullPolicy: Always
+          image: byteflair/rabbitmq-stomp
```

Comment:
I noticed that this image is missing a tag.  Best practice would be to always include it, since a future rabbit version could break things otherwise.